marnil-hot/SJ1-PR3-TheDataGenerator
# Team4 us3 genererate random email

Team 4 User story 3 generate random email.

**How the method works**
1. Put in how long random email you want.
2.Press Generate random email.
3.The result pops up in the TextArea.

**Testing**
1. Check the length of the string.
2. Try put in diffrent values in the "length of string" field.
